src/librustdoc/passes.rs

```diff
@@ -106,7 +106,7 @@ pub fn strip_private(mut krate: clean::Crate) -> plugins::PluginResult {
             retained: &mut retained,
             access_levels: &access_levels,
         };
-        krate = stripper.fold_crate(krate);
+        krate = ImportStripper.fold_crate(stripper.fold_crate(krate));
```

Comment:
Yes, I think that'd be a good idea. I'll submit a PR later if you or someone else hasn't by then.